packages/common/src/types/rollup.ts

```diff
@@ -17,18 +17,24 @@ export enum ERollUpState {
   Failed = 'Failed',
 }
 
-export type TRollUpHistory = TDbRecord<{
+export type TRollUpHistory = {
   databaseName: string;
   currentMerkleTreeRoot: string;
   previousMerkleTreeRoot: string;
   // Previous name `txId` is changed to `transactionObjectId`,
   // txId is not a good name it's alias of tx hash
   transactionObjectId: ObjectId;
   proofObjectId: ObjectId;
-  transactionHash: string;
-  status: ETransactionStatus;
-  error: string;
-}>;
+};
+
+export type TRollUpHistoryRecord = TDbRecord<TRollUpHistory>;
+
+// Compound Type
+export type TRollUpTransactionHistory = Omit<
+  TRollUpHistoryRecord,
+  'updatedAt' | '_id'
+> &
```

Comment:
Why omit `updateAt` but keep `createdAt`?

Reply:
That's what required for UI

Reply:
You can return whatever you want and front-end could choose what to display, don't think in RESTful way.